### PR TITLE
I18n sync HOC 08/23

### PR DESCRIPTION
**Pull request by deploy-code-org**

### CODE REVIEW

**Comment by bethanyaconnor** [comment]
Oh I forgot to delete the old translations so everything is duplicated...

I might need to redo the down-out to fix this.

File: i18n/locales/fr-FR/hourofcode/how-to/afterschool.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
 nav: how_to_nav
 ---
-<%= view :signup_button %>
+
+{{ signup_button }}
 
 # Comment enseigner une Heure de Code en activités para-scolaires ou en clubs
 
-### Rejoignez le mouvement et initiez un groupe d'élèves à leur première heure d'informatique grâce à ces étapes. L'Heure de Code est facile à exécuter - même pour les débutants! Si vous souhaitez de l'aide, vous pouvez trouver un [bénévole local](%= codeorg_url('/volunteer/local') %) pour vous aider à animer l'Heure de Code dans votre classe ou votre club.
+### Rejoignez le mouvement et initiez un groupe d'élèves à leur première heure d'informatique grâce à ces étapes. L'Heure de Code est facile à mettre en œuvre - même pour les débutants! If you'd like an extra set of hands to help out, you can find a [local volunteer]({{ urls/volunteer_local }}) to help run an Hour of Code in your after-school class or club.
```
Comment (nkiruka) [inline i18n/locales/fr-FR/hourofcode/how-to/afterschool.md:11]: It seems that some strings are untranslated due to the erb-partial conversion.

**Review by nkiruka — APPROVED** [review]

**Comment by Hamms** [comment]
Looks like everything ended up in `.md` files, not `.md.partial`

**Comment by bethanyaconnor** [comment]
Not on pegasus -- see 452a0a765747a4dd447446921d6956efa391b154

**Comment by Hamms** [comment]
oh, fantastic!

Looks like the test failure is legit, probably due to some invalid translations:

image

(is it new that you can't copy text from a drone log?)

**Comment by bethanyaconnor** [comment]
Those test failures are due to `layout: wide` being translated in Polish. I fixed that in the code and I'll go into Crowdin and fix there.

There were also some issues with the yml files -- see 00559ab651cad181fad6a7848d827966aa1c8a04. I'm not sure what caused them.

**Review by Hamms — APPROVED** [review]
LGTM, pending tests!

**Pull request merged**